### DISCUSSION TITLE
Not able to change page title dynamically

### DISCUSSION THREAD

**Discussion by berkturetken** [Azure/api-management-developer-portal#2607] [Feature request]
## Problem description

There is no option or way to change the webpage title (i.e., tab title) dynamically according to the clicked API from the API list. It would be great to have dynamic declaration while defining the "title" in the "Pages - Page" section (see Figure 1 below). Currently, we can only define a static "title" and that defined same title is shown as a page title for each and every API. Or any other solution in which we would be able to define specific webpage titles for different APIs is also appreciated.

The only option to achieve this is to have a self-hosted portal but it would be nice to have add this feature to the managed portal as well.

## Is your portal managed or self-hosted?

Managed

## Additional context
Figure 1:
![image](https://github.com/Azure/api-management-developer-portal/assets/29168666/aa967c7f-1454-4107-97b6-f896e6d3398e)

**Comment by ghost** [comment]
@berkturetken, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@berkturetken, thank you for requesting this feature.
